zookeeper/tf/model.py

```diff
@@ -1,15 +0,0 @@
-from abc import ABC, abstractmethod
```

Comment:
Not quite sure what to do about this. I actually can't imagine any scenario in which you wouldn't want to use `@factory` for defining a Keras model. But there's also not really a nice way to specify a base class for an `@factory`, so I've removed this for now. It's not like this was really doing anything previously anyway.